src/index.html

```diff
@@ -3,10 +3,457 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://static-00.iconduck.com/assets.00/cosmos-cryptocurrency-icon-2048x2048-k2k2lftp.png"
+    rel="icon" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
+  <header class="header">
+    <a class="logo" href="#home" ><img src="images/logo.png" alt="Logo"></a>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a class="nav__link is-active" href="#apple">Apple</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#samsung">Samsung</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#smartphones">Smartphones</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#laptops&computers"
+          data-qa="nav-hover" >Laptops & Computers</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#gadgets">Gadgets</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#tablets">Tablets</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#photo">Photo</a>
+        </li>
+        <li class="nav__item">
+          <a class="nav__link" href="#video">Video</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="catalog">
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__add-info">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+        </div>
+
+        <p class="card__review">
+          Review:5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+
+        <p class="card__price-amount">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        class="card__link"
+        href="#buy"
+        data-qa="hover"
```

Comment:
The data-qa attribute value should be 'card-hover' instead of 'hover'. Change the attribute value to 'card-hover'